gui/storage/forms.py

```diff
@@ -980,15 +980,10 @@ def _populate_disk_choices(self):
                 for i in c.call('disk.get_encrypted', {'unused': True})
             ]
 
-    def clean(self):
+    def done(self, *arg, **kwargs):
+        super(AutoImportDecryptForm, self).done()
```

Comment:
I think we may need to pass *args and **kwargs here?

Reply:
Whoops, just saw this. Fixed it.